From 1c7d629f2f0d1f9d926c880491bf001e24287404 Mon Sep 17 00:00:00 2001
From: Konrad Adamczyk <konrad.adamczyk@cavium.com>
Date: Thu, 21 Jun 2018 11:21:12 +0200
Subject: [PATCH 0781/1239] drivers: usb eth: Fix displaying the interface name

Interface name is concatenated with next string,
which causes confusion in proper interface name.

Add space to prevent above behavior.

Signed-off-by: Konrad Adamczyk <konrad.adamczyk@cavium.com>
---
 drivers/usb/eth/usb_ether.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/eth/usb_ether.c b/drivers/usb/eth/usb_ether.c
index 4a41899e86..8821f7bcc6 100644
--- a/drivers/usb/eth/usb_ether.c
+++ b/drivers/usb/eth/usb_ether.c
@@ -240,7 +240,7 @@ static void probe_valid_drivers(struct usb_device *dev)
 			if (eth_write_hwaddr(eth, "usbeth",
 					usb_max_eth_dev - 1))
 				puts("Warning: failed to set MAC address\n");
-			printf("	%s", eth->name);
+			printf("	%s ", eth->name);
 			break;
 			}
 		}
-- 
2.29.0

